src/render.js

```diff
@@ -2,7 +2,10 @@
 const fs = require('fs');
 const path = require('path');
 const { remote, ipcRenderer } = require('electron');
+const Store = require('electron-store');
+const SUPPORTED_LANGUAGES = require('../node_modules/cucumber-forge-report-generator/src/locales/supportedLocales.json');
```

Comment:
seems like something we should probably export as a first-class member of Generator instead of accessing this way, as the file could change/be moved without a major breaking change to the Generator API, and the app would be broken

Reply:
Is that something we would just define in the package.json for cucumber-forge-report-generator?

Reply:
nah, it'd just be a [static member](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields#Public_static_fields) of cucumber-forge-report-generator

Reply:
Ahh you mean like a static member of the Generator class.  Yeah that makes sense.

Reply:
it's fine to do that later. i'll approve and carry on.